labs/lab-2-javascript-game/stylesheet.css

```diff
@@ -0,0 +1,178 @@
+body {
+  background-color: rgb(47, 115, 181);
+  color: rgb(255, 255, 255);
+  margin: 0;
+  font-family: 'Titillium Web', sans-serif;
+}
+
+#page {
+  display: flex;
+  margin: 0;
+  justify-content: center;
+  align-items: center;
+  height: 600px;
+}
+
+#intro {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+}
+
+.text {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+}
+
+button {
+  width: 150px;
+  height: 50px;
+  margin: 10px;
+  font-size: 20px;
+  background: linear-gradient(rgb(79, 152, 223), rgb(56, 121, 184));
+  box-shadow: 5px 5px 20px -5px;
+  border: none;
+  border-radius: 20px;
+  width: 150px;
+}
+
+#battle {
+  margin-top: 25px;
+  display: none;
+  flex-direction: column;
+}
+
+.enemy-section {
+  display: flex;
+  flex-direction: row-reverse;
+  text-align: right;
+}
+
+.player-section {
+  display: flex;
+  flex-direction: row;
+  text-align: left;
+}
+
+.battle-info {
+  display: flex;
+  flex-direction: column;
+  width: 300px;
+}
+
+.enemy-icon, .player-icon {
+  padding: 5px;
+}
+
+img {
+  width: 150px;
+  height: 150px;
+  border-radius: 10%;
+  border: solid rgb(0, 0, 0) 5px;
+  background-color: rgb(255, 255, 255);
+  padding: 4px;
+}
+
+#enemy-name, #player-name {
+  font-size: 20px;
+}
+
+.health-bar {
+  min-width: 300px;
+  height: 20px;
+  background-color: #FFF;
+  margin: 0px;
+}
+
+.remaining-health-bar {
+  height: 20px;
+  margin: 0px;
+  background-color: rgb(61, 242, 57);
+  width: 100%;
+  transition: width 1s, background-color 1s;
```

Comment:
O la la! Very fancy.

As a rule of thumb, 1s is a LONG transition. 250ms is about as long as I ever see in production code, and 150ms or 125ms are common. Users want feedback as quickly as possible; motion can be a nice way to bring attention to what's changing, but don't stretch it too long.

You could also use steps for the transition-timing-function (https://developer.mozilla.org/en-US/docs/Web/CSS/transition-timing-function) to make it take discrete steps, implying the granular nature of health.  I dunno if that would look good or not, but it's something to experiment with.